pyproject.toml

```diff
@@ -4,7 +4,6 @@
 requires = [
     # First version of setuptools to support pyproject.toml configuration
     "setuptools>=61.0.0",
```

Comment:
Do we need to bump minimum setuptools dep?

Reply:
Unclear? Latest setuptools is 3.8+, but obviously our 3.7 tests are passing with this, which suggests it was never required?